src/lib/components/CourseSearchResults/index.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview An Algolia search box.
+ */
+
+import {html} from 'lit-element';
+import {unsafeHTML} from 'lit-html/directives/unsafe-html';
+import {SearchResults} from '../SearchResults';
+import 'focus-visible';
+import './_styles.scss';
```

Comment:
One thing worth thinking about is if we want to start moving away from this pattern of custom elements async loading their styles. It can create issues if styles are loaded in the wrong order. The other option would be to add this scss to components/all.scss.

I'm not saying we need to change it right now because I know that the existing search box uses this pattern. But I've been trying to avoid it for new components.